samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/backstack/BackStackExampleNode.kt

```diff
@@ -140,183 +144,250 @@ class BackStackExampleNode(
                         }
                     }
                 }
-                Spacer(modifier = Modifier.size(8.dp))
-                Column(
-                    modifier = Modifier.fillMaxWidth()
-                ) {
-                    Text(text = "Child: ", fontWeight = Bold)
-                    Row {
-                        listOf("A", "B", "C", "D").forEach {
-                            Row {
-                                RadioButton(
-                                    selected = it == selectedChildRadioButton.value,
-                                    enabled = isRadioButtonNeeded.value,
-                                    onClick = { selectedChildRadioButton.value = it }
-                                )
-                                Text(text = it)
-                                Spacer(modifier = Modifier.size(36.dp))
-                            }
-                        }
-                    }
+                ChildColumn(
+                    selectedChildRadioButton = selectedChildRadioButton,
+                    isRadioButtonNeeded = isRadioButtonNeeded,
+                    defaultOrRandomRadioButton = defaultOrRandomRadioButton,
+                )
+                IdColumn(
+                    selectedId = selectedId,
+                    isIdNeeded = isIdNeeded,
+                    backStackState = backStackState,
+                )
+                OperationColumn(
+                    selectedOperation = selectedOperation,
+                    selectedChildRadioButton = selectedChildRadioButton,
+                    defaultOrRandomRadioButton = defaultOrRandomRadioButton,
+                    selectedId = selectedId,
+                    isRadioButtonNeeded = isRadioButtonNeeded,
+                    isIdNeeded = isIdNeeded,
+                    areThereMissingParams = areThereMissingParams,
+                )
+                MissingParamsColumn(
+                    selectedOperation = selectedOperation,
+                    selectedChildRadioButton = selectedChildRadioButton,
+                    areThereMissingParams = areThereMissingParams,
+                    selectedId = selectedId,
+                    defaultOrRandomRadioButton = defaultOrRandomRadioButton,
+                    backStackState = backStackState,
+                )
+                BackstackColumn(backStackState = backStackState)
+                ColumnSkipRendering(skipChildRenderingByNavTarget = skipChildRenderingByNavTarget)
+            }
+        }
+    }
+
+    @Composable
+    private fun ChildColumn(
```

Comment:
Similar comment here about not having a scope (which could cause issues with the eventual Compose lint rules)